File: android/app/src/main/java/com/br/app/deliverydelicatessen/model/EstablishmentBrandDebit.kt
package com.br.app.deliverydelicatessen.model

data class EstablishmentBrandDebit(
    var active: Boolean = false,
    var establishmentId: Int = 0,
    var brandId: Int = 0,
    var id: Int = 0,
    var brand: Brand? = null)

File: financial/Models/EstablishmentBrandDebit.cs
using System.ComponentModel.DataAnnotations.Schema;
namespace Models
{
    [Table("EstablishmentBrandDebit")]

    public class EstablishmentBrandDebit : BaseEntity
    {

        public int EstablishmentId { get; set; }
        public int BrandId { get; set; }
        public bool Active { get; set; } = true;

        [NotMapped]
        public virtual Brand Brand { get; set; }

    }
}
